api/src/main/java/com/nicknathanjustin/streamercontracts/twitch/TwitchService.java

```diff
@@ -16,6 +16,14 @@
      */
     Map<String, List<Map<String, Object>>> getTopGames();
 
+    /**
+     * Get the a game from the twitch api
+     *
+     * @return A Map with 1 field, data.
+     * data contains the game
```

Comment:
Remove this line?

Reply:
This is the format that is returned from twitch, we are just forwarding it to the frontend. I don't think we should make a "game" object because this is defined by twitch.